srcpkgs/ovmf-ia32/template

```diff
@@ -0,0 +1,102 @@
+# Template file for 'ovmf-ia32'
+pkgname=ovmf-ia32
+version=201911
+revision=1
+archs=noarch
+wrksrc="edk2-edk2-stable${version}"
+hostmakedepends="make python acpica-utils nasm"
+makedepends="libuuid-devel"
+_basedesc="Tianocore UEFI firmware for qemu"
+short_desc="${_basedesc} ia32"
+maintainer="Franc[e]sco <lolisamurai@tfwno.gf>"
+license="BSD-2-Clause-Patent, MIT, Apache-2.0"
+homepage="https://github.com/tianocore/edk2/tree/master/OvmfPkg"
+_openssl_ver=1.1.1b
+distfiles="https://github.com/tianocore/edk2/archive/edk2-stable${version}.tar.gz
+https://www.openssl.org/source/openssl-${_openssl_ver}.tar.gz"
+checksum="23affd4ca2ba526747e72cbb350a4c95d192ac14eeb616778b1976577ed06001
+5c557b023230413dfb0756f3137a13e6d726838ccd1430888ad15bfb2b43ea4b"
+_toolchain_opt=GCC5
+_ovmf_common_flags="
+ -n ${XBPS_MAKEJOBS}
+ -b RELEASE
+ -D NETWORK_IP6_ENABLE
+ -D TPM2_ENABLE
+"
+_ovmf_x86_flags="
+ ${_ovmf_common_flags}
+ -D HTTP_BOOT_ENABLE
+ -D TLS_ENABLE
+ -D FD_SIZE_2MB
+"
+_ovmf_x86_sb_flags="
+ ${_ovmf_x86_flags}
+ -D SECURE_BOOT_ENABLE
+ -D SMM_REQUIRE
+ -D EXCLUDE_SHELL_FROM_FD
+"
+
+case ${XBPS_MACHINE} in
+	i686)
+		# edk2 doesn't find the correct host.
+		export HOST_ARCH=IA32 ;;
+esac
+
+post_extract() {
+	mv "${XBPS_BUILDDIR}/openssl-${_openssl_ver}/"* CryptoPkg/Library/OpensslLib/openssl/
+}
+
+do_build() {
+	export PATH="${wrksrc}/BaseTools/BinWrappers/PosixLike/:$PATH"
+	export EDK_TOOLS_PATH="${wrksrc}"/BaseTools
+	export PYTHON_COMMAND=python
+	export WORKSPACE="${wrksrc}"
+	export PACKAGES_PATH="${wrksrc}"
+	bash -c ". ./edksetup.sh"
+	make -C BaseTools
+	# x86_64, no secure boot, no smm
+	command build -t ${_toolchain_opt} \
+		-a X64 -p OvmfPkg/OvmfPkgX64.dsc ${_ovmf_x86_flags}
+	# x86_64, secure boot, smm
+	command build -t ${_toolchain_opt} \
+		-a IA32 -a X64 -p OvmfPkg/OvmfPkgIa32X64.dsc ${_ovmf_x86_sb_flags}
```

Comment:
why -a IA32  and -a X64?

Reply:
it's what fedora's spec uses to build the secboot image. not entirely sure if that's required